### PR TITLE
Tidy-up and bug fixes

### PR DESCRIPTION
**Pull request by MiguelGuthridge**
* [x] Change else if to elif
* [x] Create generic system for role management using buttons
* [x] Refactor polling code to make it less repetitive and complex
* [x] Split into multiple files to make it more maintainable
* :x: Create generic system for role management using drop-downs (will leave for later)

### CODE REVIEW

**Comment by JMTNTBANG** [comment]
Looking good so far!

File: main.py [hunk 1]
```diff
@@ -220,102 +177,70 @@ async def timezoneCallback(interaction):
...
+            elif select2.values[0] == "GMT+10:30":
+                role = client.get_guild(1030635475528056872).get_role(1045197320888787005)
+            elif select2.values[0] == "GMT+11:00":
+                role = client.get_guild(1030635475528056872).get_role(1045190353474555914)
+            elif select2.values[0] == "GMT+12:00":
+                role = client.get_guild(1030635475528056872).get_role(1045190356741935145)
+            elif select2.values[0] == "GMT+13:00":
+                role = client.get_guild(1030635475528056872).get_role(1045190359870869634)
+            elif select2.values[0] == "GMT+14:00":
+                role = client.get_guild(1030635475528056872).get_role(1045190363889016864)
```
Comment (JMTNTBANG) [inline main.py:243]: Would this fix the issues with the dropdowns not working?

File: main.py [hunk 2]
```diff
@@ -23,160 +23,117 @@
...
+            role = client.get_guild(1030635475528056872).get_role(role["role"])
+            member = client.get_guild(1030635475528056872).get_member(button_info.user.id)
+
+            if member in role.members:
+                await member.remove_roles(role)
+                await button_info.response.send_message(f'Removed "{role["name"]}" Role', ephemeral=True)
+            else:
+                await member.add_roles(role)
+                await button_info.response.send_message(role["response"], ephemeral=True)
+        return callback
```
Comment (JMTNTBANG) [inline main.py:76]: Definitely more efficient this way thank you!

File: main.py (hunk 1 above)
Comment (MiguelGuthridge) [reply main.py:243]: Not sure, but it at least makes it far nicer to read

**Comment by JMTNTBANG** [comment]
image

**Pull request merged**